lib/components/HeroCard/HeroCard.tsx

```diff
@@ -24,13 +26,24 @@ export interface HeroCardProps {
 }
 
 const Arrow = ({ color }: { color: string }) => (
-  <div className="ds-size-32 ds-ml-auto ds-select-none ds-font-bold" aria-hidden>
+  <div className="sm:ds-ml-auto" aria-hidden>
```

Comment:
In the props above, ` arrowVisible?: boolean;` has mention about `href` but it should be `to` I assume.